### PR TITLE
Assignment_Q10_Ravi

### PR DESCRIPTION
**Pull request by RaviSTechnocredits**
Calculate sum of prime numbers between user specified range

### CODE REVIEW

File: JavaAssignment/src/ravi/ClientPrimeNumberSum.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package ravi;
+
+public class ClientPrimeNumberSum {
+	public static void main(String[] args) {
+		PrimeNumberSum object = new PrimeNumberSum();
+		int start;
```
Comment (KrishnaKTechnocredits) [inline JavaAssignment/src/ravi/ClientPrimeNumberSum.java:6]: Here you are simply adding a line, rest of it you have to define start variable in for a loop.

File: JavaAssignment/src/ravi/PrimeNumberSum.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package ravi;
+
+public class PrimeNumberSum {
+	int sum = 0;
+
+	int isPrime(int number) {
```
Comment (KrishnaKTechnocredits) [inline JavaAssignment/src/ravi/PrimeNumberSum.java:6]: The method name is good.

File: JavaAssignment/src/ravi/PrimeNumberSum.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+		boolean flag = true;
+		for (int start = 2; start <= number / 2; start++) {
+
+			if (number % start == 0) {
```
Comment (KrishnaKTechnocredits) [inline JavaAssignment/src/ravi/PrimeNumberSum.java:10]: Remove extra space, add comments before a method and format your code.

**Review by KrishnaKTechnocredits — COMMENTED** [review]
Logic is correct to do formatting related changes.

**Review by mkanani13 — APPROVED** [review]
Code looks good to me

**Pull request merged**